Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "DeepDiffs"
+uuid = "ab62b9b5-e342-54a8-a765-a90f495de1a6"
+license = "MIT"
+desc = "Compute and pretty-print diffs for data structures like arrays and dictionaries"
+keywords = ["diff"]
+version = "0.1.0"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+
+[compat]
+Compat = "3"
+julia = "1"
```

Comment:
Double check for conditional code based upon the version like: https://github.com/ssfrr/DeepDiffs.jl/blob/cb5550b5e2d002285bc2341438d02afa045e37a7/src/DeepDiffs.jl#L17 as these can be removed

Reply:
The `.travis.yml` and `appveyor.yml` need to be updated to no longer test against Julia 0.6

Reply:
Addressed